### ISSUE TITLE
example for aria-sort in section 6.4 has an orphan aria-label

### ISSUE THREAD

**Issue by sasgrw** [w3c/aria-practices#674]
https://www.w3.org/TR/wai-aria-practices/#gridAndTableProperties_sort

It appears to be a formatting error.  The `aria-label` should be inside the `<table>` tag.  Currently, it's sitting on a line by itself.
